projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job.py

```diff
@@ -135,6 +136,17 @@ def get_job_execution_status(self, execution_id: str) -> DataJobExecution:
         )
         return job_execution
 
+    def get_job_executions(self) -> Optional[List[DataJobExecution]]:
```

Comment:
again I think the docs need to declare the sorting order of the returned executions.

Reply:
Updated doc. For more information, see https://github.com/vmware/versatile-data-kit/pull/1527#discussion_r1083932427